### PR TITLE
update the description related to clustered indexes

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: NA
- Other reference link(s): #5771 #5774

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: clustered-indexes.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ CREATE TABLE t (a BIGINT, b VARCHAR(255), PRIMARY KEY(a, b) NONCLUSTERED);
 
 注意，列定义中的 `KEY` 和 `PRIMARY KEY` 含义相同。
 
-此外，TiDB 支持[可执行的注释语法](/comment-syntax.md)：
+此外，TiDB 支持使用[可执行的注释语法]指定聚簇索引属性(/comment-syntax.md)：
```
Comment (TomShawn) [inline clustered-indexes.md:53]:
```suggestion
此外，TiDB 支持使用[可执行的注释语法](/comment-syntax.md)指定聚簇索引属性：
```

File: system-variables.md [hunk 2]
```diff
@@ -430,20 +430,12 @@ mysql> SELECT * FROM t1;
...
-
-    开启该特性对性能的影响主要体现在以下几个方面:
-
-    - 插入的时候每行会减少一个索引 key 的写入。
-    - 使用主键作为等值条件查询的时候，会节省一次读取请求。
-    - 使用单列主键作为范围条件查询的时候，可以节省多次读取请求。
-    - 使用多列主键的前缀作为等值或范围条件查询的时候，可以节省多次读取请求。
+- 作用域：GLOBAL
+- 默认值：INT_ONLY
+- 这个变量用于控制默认情况下表的主键是否使用[聚簇索引](/clustered-indexes.md)。“默认情况” 即不显式指定 `CLUSTERED`/`NONCLUSTERED` 关键字的情况。可设置为 `OFF`/`ON`/`INT_ONLY`。
```
Comment (TomShawn) [inline system-variables.md:435]:
```suggestion
- 这个变量用于控制默认情况下表的主键是否使用[聚簇索引](/clustered-indexes.md)。“默认情况”即不显式指定 `CLUSTERED`/`NONCLUSTERED` 关键字的情况。可设置为 `OFF`/`ON`/`INT_ONLY`。
```

**Comment by TomShawn** [comment]
Rest /lgtm

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 
 > **注意：**
 >
-> 该配置项不再生效。如果需要增删主键，请使用 `NONCLUSTERED` 代替。要了解关于 `CLUSTERED` 主键的详细信息，请参考[聚簇索引](/clustered-indexes.md)。
+> 该配置项已被废弃，目前仅在 `@@tidb_enable_clustered_index` 取值为 `INT_ONLY` 时生效。如果需要增删主键，请在建表时使用 `NONCLUSTERED` 关键字代替。要了解关于 `CLUSTERED` 主键的详细信息，请参考[聚簇索引](/clustered-indexes.md)。
```
Comment (TomShawn) [inline tidb-configuration-file.md:101]: This line has been translated by @Joyinqin in https://github.com/pingcap/docs/pull/5105.

File: br/backup-and-restore-tool.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ BR 和 TiDB 集群的兼容性问题分为以下两方面：
 
 | 功能 | 相关 issue | 解决方式 |
 |  ----  | ----  | ----- |
-| 聚簇索引 | [#565](https://github.com/pingcap/br/issues/565)       | 确保备份时 tidb_enable_clustered_index 变量和恢复时一致，否则会导致数据不一致的问题，例如 `default not found` 和数据索引不一致。 |
+| 聚簇索引 | [#565](https://github.com/pingcap/br/issues/565)       | 确保备份时 tidb_enable_clustered_index 全局变量和恢复时一致，否则会导致数据不一致的问题，例如 `default not found` 和数据索引不一致。 |
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:79]: This line is assigned to @TomShawn to translate together with https://github.com/pingcap/docs-cn/pull/5224.

**Pull request merged**